benchmark/Streamly/Benchmark/Data/Stream/Eliminate.hs

```diff
@@ -453,23 +488,25 @@ o_1_space_elimination_folds value =
         , benchIOSink value "maximum" maximum
         , benchIOSink value "minimumBy" minimumBy
         , benchIOSink value "minimum" minimum
-
+#ifdef USE_PRELUDE
         , bench "the" $ nfIO $ randomRIO (1,1) >>= the . repeat value
+#endif
         , benchIOSink value "find" (find value)
         , benchIOSink value "findM" (findM value)
-        -- , benchIOSink value "lookupFirst" (lookup 1)
+        , benchIOSink value "lookupFirst" (lookup 1)
         , benchIOSink value "lookupNever" (lookup (value + 1))
         , benchIOSink value "(!!)" (value !!)
         , benchIOSink value "findIndex" (findIndex value)
         , benchIOSink value "elemIndex" (elemIndex value)
         -- this is too fast, causes all benchmarks reported in ns
-    -- , benchIOSink value "null" S.null
+        , benchIOSink value "null" S.null
```

Comment:
Benchmarks is too small, should not uncomment this. See the comment before this which tells why this was commented.